autoload/lsp/ui/vim/output.vim

```diff
@@ -112,15 +112,8 @@ function! lsp#ui#vim#output#floatingpreview(data) abort
     endif
 
     let l:opts = s:get_float_positioning(l:height, l:width)
-
-    let s:winid = nvim_open_win(buf, v:true, l:opts)
-    call nvim_win_set_option(s:winid, 'winhl', 'Normal:Pmenu,NormalNC:Pmenu')
-    call nvim_win_set_option(s:winid, 'foldenable', v:false)
-    call nvim_win_set_option(s:winid, 'wrap', v:true)
-    call nvim_win_set_option(s:winid, 'statusline', '')
-    call nvim_win_set_option(s:winid, 'number', v:false)
-    call nvim_win_set_option(s:winid, 'relativenumber', v:false)
-    call nvim_win_set_option(s:winid, 'cursorline', v:false)
+    let l:opts['style'] = 'minimal'
+    let s:winid = nvim_open_win(buf, v:false, l:opts)
     " Enable closing the preview with esc, but map only in the scratch buffer
     nmap <buffer><silent> <esc> :pclose<cr>
```

Comment:
Now that we're not focusing the float, this would apply to the main buffer, I believe.

Reply:
Ah, you're right. Didn't notice that. How weird...